0.wave1_data/3.process_features/3.process_features.yml

```diff
@@ -0,0 +1,18 @@
+name: 3.process_features_interstellar
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.8.13
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::jupyterlab
+  - conda-forge::pandas=1.4.2
+  - conda-forge::plotnine
+  - conda-forge::ipykernel
+  - conda-forge::nb_conda_kernels
+  - conda-forge::jupyter_contrib_nbextensions
+  - conda-forge::scipy=1.7
+  - conda-forge::numpy=1.22
+  - conda-forge::scikit-learn
+  - conda-forge::sqlalchemy=1.4.42
```

Comment:
Would it make sense to provide a comment here as to the static version reference for possible later revision?

Reply:
That would be good! Ill add in the comment!